README.md

```diff
@@ -23,7 +27,7 @@ and run `pod install` from the main directory.
 
 ## Getting Started
 
-To get started, first, import the header file DKDBManager.h in your project's pch file. This will import in your project all required headers for `DKHelpers`, `Magical Record` and CoredData.
+To get started, first, import the header file DKDBManager.h in your project's .pch or bridge-header file. This will import in your project all required headers for `DKHelpers`, `Magical Record` and `CoredData`.
```

Comment:
this sentence needs to be improved ... maybe: `This will import all required headers in your project (DKHelpers, Magical Record and CoredData)` ?!